src/components/NavigationIcon/NavigationIcon.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './NavigationIcon.scss';
+import { Avatar, SvgIconTypeMap } from '@material-ui/core';
+import { Link, useLocation } from 'react-router-dom';
+import { OverridableComponent } from '@material-ui/core/OverridableComponent';
+
+interface NavigationIconProps {
+  path?: string | null;
+  avatar?: string | undefined;
+  Icon?: OverridableComponent<SvgIconTypeMap<unknown, 'svg'>> | null;
+  title: string;
+}
+
+export const NavigationIcon: React.FC<NavigationIconProps> = ({ path, avatar, Icon, title }) => {
+  const location = useLocation();
+
+  if (path) {
+    return (
+      <Link to={path}>
+        <div className={`navigationIcon ${path === location.pathname ? 'navigationIcon--active' : ''}`}>
```

Comment:
I think is too much

Reply:
https://www.npmjs.com/package/@mateusz_lisowski/merge-classnames

cx([{ "navigationIcon--active": path === location.pathname }, "fixed-class"]);